persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -323,6 +313,39 @@ LEFT JOIN MessageConversationChangedContent AS ConversationNameChangedContent ON
 LEFT JOIN MessageAssetContent AS AssetContent ON Message.id = AssetContent.message_id AND Message.conversation_id = AssetContent.conversation_id
 LEFT JOIN MessageTextContent AS TextContent ON Message.id = TextContent.message_id AND Message.conversation_id = TextContent.conversation_id;
 
+needsToBeNotified:
```

Comment:
is this part of the logic covered with unit tests

Reply:
Will add in another PR.